src/components/screens/multiSignature/editor/editor.js

```diff
@@ -144,10 +185,12 @@ const Editor = ({
           priorityOptions={priorityOptions}
           selectedPriority={selectedPriority.selectedIndex}
           setSelectedPriority={selectTransactionPriority}
+          loadError={prioritiesLoadError}
+          isLoading={loadingPriorities}
         />
-        {
+        {/* {
           feedback.error && <span className="feedback">{feedback.messages[0]}</span>
-        }
+        } */}
```

Comment:
Should this not be uncommented?

Reply:
I will work on the validation in a separate ticket.